web/client/product/app.jsx

```diff
@@ -10,4 +10,8 @@ import main from './main';
 import appConfig from './appConfig';
 import pluginsDef from './plugins';
 
+
+console.log("plugin not correctly loaded: ", Object.keys(pluginsDef.plugins).filter(plugin => pluginsDef.plugins[plugin].default));
```

Comment:
Is this for identifying plugins required without the default?

Reply:
maybe we can check if the list is not empty, and log only in that case (and maybe also log this as an error?) What do you think @MV88 ?

Reply:
yes, i thought it was good to have a console message for it